### DISCUSSION TITLE
how to cite mirdata

### DISCUSSION THREAD

**Discussion by PRamoneda** [mir-dataset-loaders/mirdata#425] [Q&A] [answered]
Hi everyone!

If I want to cite mirdata (because I am using the loaders for X publication). Which DOI should I use?
- DOI to all the versions (https://doi.org/10.5281/zenodo.4355858)
>> Mirdata [Computer software]. (2019). Retrieved from https://doi.org/10.5281/zenodo.4355859.
- DOI to only the last version (https://doi.org/10.5281/zenodo.4355859)
>> Magdalena Fuentes; Rachel Bittner; Marius Miron; Genís Plaja; Pedro Ramoneda; Vincent Lostanlen; David Rubinstein; Andreas Jansson; Thor Kell; Keunwoo Choi; Tom Xi; Kyungyun Lee; Xavier Serra (2020). Mirdata v.3.0.0 [Computer software]. Washington, DC: E & K Press. DOI:10.5281/zenodo.4355859

**Comment by magdalenafuentes** [comment, marked as answer]
If you are using 0.3.0 it’s better that you cite the later (4355859), to know exactly what version of the loaders you used (for reproducibility).
